wheelhouse.txt

```diff
@@ -3,7 +3,18 @@ gunicorn>=20.0.0,<21.0.0
 loadbalancer-interface
 typing_extensions<4.0
 
+# psutil 5.9.3 starts requirement of setuptools>=43
+# either we use this
+# psutil<5.9.3
+# or we use the default install in focal
+# setuptools>=45.2.0
+setuptools>=45.2.0
```

Comment:
We should really improve the comment here to explain WHY we're upgrading the setuptools.  I think it links nicely with the update of pip below as well.